zebra/if_netlink.c

```diff
@@ -794,8 +800,6 @@ static int netlink_bridge_interface(struct nlmsghdr *h, int len, ns_id_t ns_id,
 
 	/* We are only interested in the access VLAN i.e., AF_SPEC */
 	af_spec = tb[IFLA_AF_SPEC];
-	if (!af_spec)
```

Comment:
hmm - below this, `netlink_bridge_vlan_update()` is called with `af_spec` - which could now be NULL. It looks like that may not be safe?

Reply:
I also changed the `netlink_bridge_vlan_update()`, the `af_spec` was only be used when it is not NULL

Reply:
quite right - my apologies - I mis-read the diff here on the webpage.